### ISSUE TITLE
Questions on USMLE questions

### ISSUE THREAD

**Issue by hyesunyun** [kbressem/medAlpaca#34]
Hello!

I had some questions about USMLE dataset that was used for evaluation.
Based on the paper, it looks like you also use the MedQA - USMLE and also additional USMLE questions for training the model.

For the evaluation, do you check to make sure that the questions in the evaluation set (test set) are not duplicates of the training set?

Also, where could I download the USMLE step questions for the evaluation? 
I see that this one is available on HuggingFace: https://huggingface.co/datasets/medalpaca/medical_meadow_usmle_self_assessment
However, it seems like this might be for training and not evaluating.

I would like to download the same version of the USMLE that you have used for the eval.

Thanks!

**Comment by kbressem** [comment]
The USMLE self assessment on HF is the correct one and was not used in training.
